frappe/core/doctype/prepared_report/prepared_report.json

```diff
@@ -84,13 +85,15 @@
    "label": "Filter Values"
   },
   {
-   "fieldname": "column_break_5",
-   "fieldtype": "Column Break"
+   "fieldname": "job_id",
+   "fieldtype": "Data",
```

Comment:
my only concern with this is, since jobs don't stay for long in rq registries, so the links will die out after sometime and might lead to weird link validation errors :thinking:

Reply:
Nope. This document isn't saved, nor is the field editable so it should be fine IMO.